vm/src/stdlib/platform.rs

```diff
@@ -1,39 +0,0 @@
-use crate::pyobject::PyObjectRef;
-use crate::version;
-use crate::vm::VirtualMachine;
-
-pub fn make_module(vm: &VirtualMachine) -> PyObjectRef {
-    let ctx = &vm.ctx;
-    py_module!(vm, "platform", {
```

Comment:
Instead of deletion, I would prefer to keep this file and rename it to `_platform`. The python code of the platform module is a bit messy, so it might be good to selectively copy paste it from the cpython sourcetree. What do you think?